aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -1346,6 +1356,7 @@ def main(args=None, *, prog="pg_migrate"):
     pg_mig = PGMigrate(
         source_conn_info=args.source,
         target_conn_info=args.target,
+        pgbin=args.pgbin,
```

Comment:
this is going to pass a str instead of a Path into the function.

Reply:
> this is going to pass a str instead of a Path into the constructor.
@alanfranz-at-work I updated the code to pass str into Path here:
https://github.com/aiven/aiven-db-migrate/blob/f21711f968035abccc876b07c1f818badcc27e0b/aiven_db_migrate/migrate/pgmigrate.py#L1185